.travis.yml

```diff
@@ -1,19 +1,27 @@
 language: go
 
 go:
-    - 1.6
-    - 1.7
-    - tip
+    - 1.9
 
 services:
     - mongodb
     - redis-server
 
-before_install:
-    mkdir $GOPATH/bin
+# comment these because of:
+# "The command "mkdir $GOPATH/bin" failed and exited with 1 during ."
+# before_install:
+#   mkdir $GOPATH/bin
+
+# comment these because of:
+# Sorry, we dont have a dist for your system: linux amd64
+# You can ask one here: https://github.com/Masterminds/glide/issues
+# Fail to install glide.
+# install:
+#     curl https://glide.sh/get | sh
```

Comment:
@hiveminded thanks for the contribution.

Failed to install glide should be caused by github api's [rate limits](https://developer.github.com/v3/rate_limit/), this issue https://github.com/Masterminds/glide/issues/639 has some solutions.
```yaml
addons:
  apt:
    sources:
      - sourceline: 'ppa:masterminds/glide'
    packages:
      - glide
```
or
```yaml
before_install:
  - sudo add-apt-repository ppa:masterminds/glide -y
  - sudo apt-get update -q
  - sudo apt-get install glide -y
```
Once again thanks for your wonderful work.